x-pack/plugins/monitoring/server/lib/alerts/fetch_cluster_health.test.ts

```diff
@@ -55,10 +55,15 @@ describe('fetchClusterHealth', () => {
   });
   it('should call ES with correct query', async () => {
     const esClient = elasticsearchClientMock.createScopedClusterClient().asCurrentUser;
-    await fetchClusterHealth(esClient, [
-      { clusterUuid: '1', clusterName: 'foo1' },
-      { clusterUuid: '2', clusterName: 'foo2' },
-    ]);
+    await fetchClusterHealth(
+      esClient,
+      [
+        { clusterUuid: '1', clusterName: 'foo1' },
+        { clusterUuid: '2', clusterName: 'foo2' },
+      ],
+      void 0,
```

Comment:
What do `void 0` do and how does it compare to null or empty string?

Reply:
That value needs to be either a `string` or `undefined`, `void 0` returns `undefined` that's compatible with older browsers; `undefined` was keyword in older browsers that could be overridden. It's probably not as important now days but old habits die hard. I can probably safely change it to `undefined` since this is server code running in Node.js.